whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/WhoisSlaveDao.java

```diff
@@ -0,0 +1,33 @@
+package net.ripe.db.nrtm4.persist;
+
+import net.ripe.db.whois.common.dao.jdbc.JdbcVersionDao;
+import net.ripe.db.whois.common.dao.jdbc.SerialRpslObjectTuple;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+
+@Service
+public class WhoisSlaveDao {
+
```

Comment:
Name is very generic and seems like connecting to readOnlyWhois datasource but its actually versionDoa and should not be in nrtm module?

Reply:
the objects it gets and the way they are sorted are specific for NRTM. i think the name could be better, though, it's a hangover from when it did sth a bit different but now it's not good. i'll change the name.